packages/aws-cdk-lib/aws-codebuild/test/linux-arm-lambda-build-image.test.ts

```diff
@@ -0,0 +1,178 @@
+import { Template } from '../../assertions';
+import * as codecommit from '../../aws-codecommit';
+import * as cdk from '../../core';
+import * as codebuild from '../lib';
+
+describe('Linux ARM Lambda build image', () => {
```

Comment:
Since there are no changes to the test content, I've written the tests only in Node.js Image.
If it looks like I should add other Images, I will add tests using jest.each, etc.